html/datasets.html

```diff
@@ -0,0 +1,308 @@
+    <!--<link rel="stylesheet" href="facetview2/css/facetview.css">-->
+    <!--<link rel="stylesheet" href="browsers.css">-->
+
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/vendor/jquery/1.7.1/jquery-1.7.1.min.js"></script>
+    <link rel="stylesheet" href="/vivo/themes/dco/js/facetview2/vendor/bootstrap/css/bootstrap.min.css">
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/vendor/bootstrap/js/bootstrap.min.js"></script>
+    <link rel="stylesheet" href="/vivo/themes/dco/js/facetview2/vendor/jquery-ui-1.8.18.custom/jquery-ui-1.8.18.custom.css">
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/vendor/jquery-ui-1.8.18.custom/jquery-ui-1.8.18.custom.min.js"></script>
+
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/es.js"></script>
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/bootstrap2.facetview.theme.js"></script>
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/jquery.facetview2.js"></script>
+    <link rel="stylesheet" href="/vivo/themes/dco/js/facetview2/css/facetview.css">
+    <link rel="stylesheet" href="/vivo/themes/dco/css/browsers.css">
+
+
+    <script type="text/javascript">
+        jQuery(document).ready(function($) {
+            $('.facet-view-simple').facetview({
+//                search_url: '//localhost:9200/dco/dataset/_search',
+                search_url: '//dcotest.tw.rpi.edu/search/dco/dataset/_search',
+                page_size: 150,
+                sort: [
+                    {"_score" : {"order" : "desc"}}
+                ],
+                sharesave_link: true,
+                search_button: true,
+                default_freetext_fuzzify: "*",
+                default_facet_operator: "AND",
+                default_facet_order: "count",
+                default_facet_size: 15,
+                facets: [
+                    /*
+                    Facets to be used are listed below (in real order):
+                        Facet Name      |       Variable Name
+                    =======================================================
+                    Type                |   mostSpecificType
+                    Publication Year    |   publicationYear
+                    ---------------------------------------------
+                    Author              |   authors.name.exact
+                    Author Organization |   authors.organization.name.exact
+                    ---------------------------------------------
+                    DCO Community       |   dcoCommunities.name.exact
+                    Portal Group        |   portalGroups.name.exact
+                    Project             |   projects.name.exact
+                    ---------------------------------------------
+                    Data Type           |   dataTypes.name.exact
+                    =======================================================
+                    */
```

Comment:
Comments should describe what's going on, what's intended, what variables/fields mean, etc... So this comment is a replication of the code and is not necessary. BUT ... the comment should be replaced with something like.

Type - defined by mostSpecificType and represents the type of the dataset being represented
Publication year - defined by publicationYear and represents the year in which the publication was published and made available for access
Author - defined by authors.name.exact meaning that the authors names are displayed in full and not separated into first, last, etc...
Author Organization - defined by authors.organization.name.exact meaning that the organization names are displayed in full and not separated into indifidual words - represents the organization that the author worked at the time of the publication of the dataset
DCO Community - dcoCommunities.name.exact meaning that the community names are displayed in full and not separated into individual words - represents one or more of the DCO science communities that a dataset is related to
Portal Group - portalGroups.name.exact meaning that the group names are displayed in full and not separated into individual words - represents one or more of the DCO teams that a dataset is related to
Project - projects.name.exact meaning that the project names are displayed in full and not separated into individual words - represents one or more projects that either generated or uses the dataset
Data Type - dataTypes.name.exact meaning that the data type names are displayed in full and not separated into individual words - represents one or more of the RDA Data Types that this dataset contains

Why replicate the information (meaning of name.exact) ... if I'm looking for a single field and what it represents I find it in a comment here, not knowing that name.exact is clarified in a separate comment but in this comment.

The more information we can provide the better.